.web.yml

```diff
@@ -8,26 +8,26 @@ tutorials:
     notebook: spam/02_spam_data_augmentation_tutorial.ipynb
     exclude_output: true
   - title: Introduction to Slicing Functions
-    description: Monitoring models for spam classification
+    description: Monitoring critical data subsets for spam classification
     notebook: spam/03_spam_data_slicing_tutorial.ipynb
     exclude_output: true
   - title: Hybrid Crowd Labeling Workflows in Snorkel
-    description: Programmatic and crowdworker labels for sentiment analysis
+    description: Mixing programmatic and crowdworker labels for sentiment analysis
     notebook: crowdsourcing
     exclude_output: true
   - title: Building Recommender Systems in Snorkel
     description: Labeling text reviews for book recommendations
     notebook: recsys
     exclude_output: true
   - title: Visual Relation Detection in Snorkel
-    description: Labeling visual relationships in images
+    description: Multi-class labeling for visual relationships in images
     notebook: visual_relation
     exclude_output: true
   - title: Information Extraction in Snorkel
-    description: Labeling spouse mentions in sentences
+    description: Labeling spouse mentions in documents
     notebook: spouse
     exclude_output: true
   - title: Introduction to Snorkel's Multitask Learning System
-    description: State-of-the-art pretraining and task flows
+    description: State-of-the-art framework for multitask learning (i.e. pretraining, parameter sharing)
```

Comment:
again for website, leaving these short and sweet looks better on the landing page, but we can try to see how to fix that

Reply:
updated !